css/_uswds-theme-custom-styles.scss

```diff
@@ -37,6 +37,26 @@ $site-measure: 5;
   @include usa-grid-col;
 }
 
+// Specific grid-col styes added for use in the card component grid
+@media all and (min-width: 55em){
+  .tablet-lg\:grid-col-6 {
+    flex: 0 1 auto!important;
+    width: 50%!important;
+  }
+}
+
+@media all and (min-width: 87.5em){
+  .widescreen\:grid-col-4 {
+    flex: 0 1 auto!important;
+    width:33.3333333333%!important;
+  }
+
+  .widescreen\:grid-col-6 {
+    flex: 0 1 auto!important;
+    width: 50%!important;
+  }
+}
+
```

Comment:
Might be more straightforward to follow a similar pattern to Banner and use the replace filter to change the class.

https://github.com/uswds/uswds-site/blob/1fec7d4889b34d0be1ed55fd9f5c10b157b06be9/_includes/code/components/banner.html#L2-L8

**Example `/_includes/code/components/card.html`**

```liquid
{% capture card--flag %}
  {% library_component usa-card~flag %}
{% endcapture %}{{ card--flag
  | strip
  | replace: 'desktop:grid-col-6', 'widescreen:grid-col-6'
}}
```

**Gives us expected result:**
<img width="900" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/fc23e6ff-a601-4153-bd0d-b896b0f5ae6e">

Reply:
Woah that's an interesting alternative! Helps the solution to stay site specific which is nice.

Reply:
@mejiaj Thanks for that suggestion! Much cleaner. I updated the templates in [c6dd23d](https://github.com/uswds/uswds-site/pull/2299/commits/c6dd23da7a445ad3ff4aec32e577b05a4451153d)

Reply:
Do we need the styles since its in the example container and that's a fixed width?

Reply:
@mahoneycm noted that if we don't want to change the Sass to bring in `widescreen` and `tablet-lg` breakpoints, we can use this code instead:
```html
<h3 class="site-preview-heading">Default card</h3>
{% capture card--standard %}
  {% library_component usa-card~standard %}
{% endcapture %}{{ card--standard
  | strip
  | replace: 'tablet:grid-col-4', 'tablet:grid-col-6 desktop-lg:grid-col-4'
}}

<h3 class="site-preview-heading">Flag layout card</h3>
{% capture card--flag %}
  {% library_component usa-card~flag %}
{% endcapture %}{{ card--flag
  | strip
  | replace: 'desktop:grid-col-6', 'flex-1'
}}
```

The breakpoints aren't quite as clean visually, but it is a noticeable improvement. This code prevents the need to either duplicate USWDS style rules in our project Sass OR add a bunch of extra CSS by turning on the two breakpoints via settings.

My personal preference is still to bring in the three grid style rules to our Sass. I know it is not ideal for code standards but for the best visual presentation I want to optimize the breakpoints as much as possible. I'd also like to see a possible feature request to be able to turn on breakpoints specifically for grid (or possibly turn it on for each individual utility), so this could be considered a stopgap measure if we decide to go down that path in the future. @mejiaj and @mahoneycm curious what you think!

Reply:
I noted my decisions on this topic in [this comment](https://github.com/uswds/uswds-site/pull/2299#issuecomment-1819706789) for visibility.